CHANGELOG.md

```diff
@@ -28,6 +28,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   Firecracker snapshots (up to Firecracker version v1.6.0) incompatible with
   the current Firecracker version.
 
+### Fixed
+
+- [#4409](https://github.com/firecracker-microvm/firecracker/pull/4409):
+  Fixed a bug in the cpu-template-helper that made it panic during conversion
+  of cpu configuration with SVE registers to the cpu template on aarch64 platform.
+  Now cpu-template-helper will print warnings if it encounters SVE registers
+  during the conversion process.
```

Comment:
nit: we might want to mention that it ignores those registers because the CPU template feature itself doesn't support the wide registers.

Reply:
added